articles/quickstarts/qrng.md

```diff
@@ -23,6 +23,7 @@ A simple example of a quantum algorithm written in Q# is a quantum random number
 As mentioned in our [What is Quantum Computing?](xref:microsoft.quantum.overview.what) article, a qubit is a unit of quantum information that can be in superposition. When measured, a qubit can only be either 0 or 1, however during execution, the state of the qubit represents the probability of reading either a 0 or a 1 during measurement. This probabilistic state is known as superposition. We can use this probability to generate random numbers.
 
 ### Q# operation code
+Let's see how we can implement the concepts showed above to create a quantum random number generator in Q#:
 
 1. Replace the contents of the Operation.qs file with the following code:
 
```

Comment:
I recognize that this isn't part of the changes you've made with this PR, but the source code at lines 32--42 seem like they have some indenting issues?